tethys_cli/install_commands.py

```diff
@@ -462,8 +468,14 @@ def install_command(args):
                 write_warning("Skipping package installation.")
             else:
                 if validate_schema('conda', requirements_config):  # noqa: E501
-                    conda_config = requirements_config['conda']
-                    install_packages(conda_config, update_installed=args.update_installed)
+                    if has_conda:
+                        conda_config = requirements_config['conda']
+                        install_packages(conda_config, update_installed=args.update_installed)
```

Comment:
Have you looked at what it would take to refactor `install_packages()` to fallback to installing the conda packages with pip?

Reply:
I didn't look into that, but it's a good suggestion. I know there are many cases were packages don't have the same name on PyPi and Conda. And of course there are some packages on Conda that wouldn't be on PyPI (e.g. postgresql). But it probably would work in many cases, so it's worth trying.